Analysis/Tasks/PWGLF/lambdakzerobuilder.cxx

```diff
@@ -73,6 +74,19 @@ DECLARE_SOA_TABLE(V0GoodIndices, "AOD", "V0GOODINDICES", o2::soa::Index<>,
 } // namespace o2::aod
 
 using FullTracksExt = soa::Join<aod::FullTracks, aod::TracksExtended>;
+using FullTracksExtMC = soa::Join<FullTracksExt, aod::McTrackLabels>;
+
+//#define MY_DEBUG
+#ifdef MY_DEBUG
+using MyTracks = FullTracksExtMC;
+#define MY_DEBUG_MSG(condition, cmd) \
```

Comment:
There is `LOG_IF(severity, condition)` which you can use. Something like:

LOF_IF(error, a < b) << "a < b!!!!!";

Reply:
Are you suggesting to replace 
```
#define MY_DEBUG_MSG(condition, cmd)
```
with 
```
#define LOG_IF(severity, condition)
```
Because in my implementation (suggested by David, to be honest), the MY_DEBUG_MSG changes behaviour according to whether there is or not the MY_DEBUG defined.